diagnostic.md

```diff
@@ -32,6 +36,13 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
+Getters
+.html = returns html tags
+.find = returns the descendants of a node
+
+Setters
+.val = will set a specified value
+.attr = will create a something new within the html, like a class
```

Comment:
Not specific enough. Also, `.find` is not technically a getter. You would need a getter after that such as these:
```
.html()
.text()
```

For setters, you can't set something without telling it what to set. 
```
.val('some data')
.attr('<some attribute>', '<some value>')
```